components/elm/cime_config/testdefs/testmods_dirs/elm/solar_rad/shell_commands

```diff
@@ -0,0 +1 @@
+./xmlchange NTASKS=48
```

Comment:
@daleihao is there a reason to set NTASKS=48 ? With this, `ERS.f09_f09.IELM.ascent_pgi.elm-solar_rad` is erroring out in 2nd/restart run with memory corruption
```
2: 14: 0: ALLOCATE: 2356269769344 bytes requested; not enough memory
2: 42: 0: ALLOCATE: 5442822862464 bytes requested; not enough memory
```
Changing NTASKS to default 1-node with 84 NTASKS lets both initial and restart runs complete BFB. Okay to remove this mod?

Reply:
It doesn't matter what  NTASKs is. It is OK to remove it.